quiz.css

```diff
@@ -0,0 +1,51 @@
+
+/*CSS FOR MAKNG YOUR TREE GROW!*/
+/*Because I really don't like plain ol' HTML*/
+
+* {
```

Comment:
Since these properties are all for text, it's better to put them onto the ```body``` tag, as that applies to all the text, and the * applies to EVERYTHING, which is a much bigger scope than necessary.